### PR TITLE
Add new TCS for TV and Iot profiles

### PR DESCRIPTION
**Pull request by sgchoi5**
For TV, update newer version (0.97 beta)
For Iot, add latest version (0.97 beta)

### CODE REVIEW

**Review by surbhig12 — APPROVED** [review]
No changes. Approving the PR.

File: docs/platform/compliance/compliance-specification.md [hunk 1]
```diff
@@ -2,30 +2,32 @@
...
   [Tizen 3.0 Compliance Specification for IVI Profile](media/tizen-3.0-compliance-specification-for-ivi-profile-v1.0_1.pdf) ![PDF icon](media/application-pdf.png)
 
   [Tizen 3.0 Crosswalk Web Runtime API for IVI Profile](media/tizen-3.0-crosswalk-apis-for_ivi-v1.0_0.pdf) ![PDF icon](media/application-pdf.png)
 
-- The following specification is the official version for Tizen 3.0 devices:
+- Tizen 3.0 devices:
 
   [Tizen 3.0 Web Runtime Core Specification](media/tizen-3.0-wrt-core-spec.pdf) ![PDF icon](media/application-pdf.png)
 
+- Mobile profile
```
Comment (surbhig12) [inline docs/platform/compliance/compliance-specification.md:25]: To maintain consistency, please add colon:

- Mobile profile:

File: docs/platform/compliance/compliance-specification.md [hunk 2]
```diff
@@ -2,30 +2,32 @@
...
+
+  [Tizen 2.4 Compliance Specification for Mobile Profile](media/tizen-2.4-compliance-specification-for-mobile-profile-v1.0.pdf) ![PDF icon](media/application-pdf.png)
+
+- Wearable profile
```
Comment (surbhig12) [inline docs/platform/compliance/compliance-specification.md:29]: To maintain consistency, please add colon:

- Wearable profile:

File: docs/platform/compliance/compliance-specification.md [hunk 3]
```diff
@@ -2,30 +2,32 @@
 
 The Tizen Compliance Specification is the definitive set of requirements for software and hardware to create compliant Tizen devices. This document is the basis for Tizen Compliance Tests and eventual Tizen Compliance certification:
 
-- The following specification is the official version for Tizen Mobile profile 2.4 devices:
+- Iot profile
```
Comment (surbhig12) [inline docs/platform/compliance/compliance-specification.md:5]: To maintain consistency, please add colon:

- IoT profile:

File: docs/platform/compliance/compliance-specification.md [hunk 4]
```diff
@@ -2,30 +2,32 @@
...
 
-  [Tizen 2.4 Compliance Specification for Mobile Profile](media/tizen-2.4-compliance-specification-for-mobile-profile-v1.0.pdf) ![PDF icon](media/application-pdf.png)
-
-- The following specification is the official version for Tizen Wearable profile 2.3.1 devices:
+  [Tizen 6.0 Compliance Specification for Iot Profile](media/tizen-6.0-compliance-specification-for-iot-profile-v0.97-beta.pdf) ![PDF icon](media/application-pdf.png)
```
Comment (surbhig12) [inline docs/platform/compliance/compliance-specification.md:7]: Please change Iot to IoT

**Review by surbhig12 — COMMENTED** [review]
Minor changes, please verify.

**Review by surbhig12 — APPROVED** [review]
Thank you for incorporating all th comments. Approving the PR.

File: docs/platform/compliance/compliance-specification.md (hunk 1 above)
Comment (sgchoi5) [reply docs/platform/compliance/compliance-specification.md:25]: Removed all ":" for consistency

File: docs/platform/compliance/compliance-specification.md (hunk 2 above)
Comment (sgchoi5) [reply docs/platform/compliance/compliance-specification.md:29]: Removed all ":" for consistency

File: docs/platform/compliance/compliance-specification.md (hunk 4 above)
Comment (sgchoi5) [reply docs/platform/compliance/compliance-specification.md]: Changed!

**Comment by sgchoi5** [comment]
Fixed all issues. plz, review it again.

**Pull request merged**